tests/TestCompleteOrder.py

```diff
@@ -7,19 +7,36 @@
 
 class TestCompleteOrder(unittest.TestCase):
 
+    """
+    TestCompleteOrder tests the Order is 'complete' with the appropriate attributes.
+
+    Method List:
+    -test_order_is_complete
+
+    Argument List:  unittest.TestCase gives the unittest model knowledge on what to test.
+
+    Author: Whitney Cormack, Temporary Oysters
+    """
+
     @classmethod
     def setUp(self):
+        """
+        Sets up a Test User, "Joey", so we can test if our code is behaving as expected.
+        """
         self.joey = customer.Customer('Joey', '787 East Silver St', 'Lebanon', 'Ohio', '35622', '5551231234')
         self.joey_mastercard = payment_options.PaymentType("Mastercard", "acct1234")
         self.joeys_cart = shoppingcart.ShoppingCart("joey")
         self.puppies = products.Product("puppies", 3)
         self.joeys_cart.add_to_cart(self.puppies)
 
 
-# test order is complete
     def test_order_is_complete(self):
-      # customer, payment type, products, total
+        """
+        Tests that our user has created an order with the correct amount of attributes and have value.
```

Comment:
How about
"Tests that our user has created an order with the correct amount of attributes with assigned values"